hello/srv/world.ts

```diff
@@ -1,5 +1,7 @@
+import { Request } from "@sap/cds/apis/services"
```

Comment:
What do you think about using `import type { Request } from "@sap/cds/apis/services`?

Reply:
Hi @tsteckenborn 
Thanks for the feedback, yes it makes sense! 
Done with 50ae4d8